build.sbt

```diff
@@ -292,7 +292,12 @@ lazy val root = (project in file("."))
         // See https://github.com/informalsystems/apalache/pull/1382
         (s"tar zxvf ${pkg} -C ${target_dir}" !)
         log.info(s"Symlinking ${current_pkg} -> ${unzipped}")
-        s"ln -sfn ${unzipped} ${current_pkg}" ! log
+        val target_path: java.nio.file.Path = java.nio.file.Paths.get(unzipped)
+        if (current_pkg.exists) {
+          log.info(s"${current_pkg} already exists, overriding")
+          current_pkg.delete
+        }
+        java.nio.file.Files.createSymbolicLink(current_pkg.toPath, target_path)
```

Comment:
I wonder if we could use the [File API exposed by sbt](https://www.scala-sbt.org/1.x/docs/Paths.html#Constructing+a+File) to `file(unzipped).toPath`?
Looks a bit more idiomatic sbt and symmetrical to the treatment of `current_pkg` below.